ow_lander/scripts/path_planning_commander.py

```diff
@@ -25,59 +25,81 @@
 import activity_deliver_sample
 import activity_grind
 
-# === MAIN COMMANDER CLASS =============================
-class MoveGroupPythonInteface(object):
+class PathPlanningCommander(object):
   
   def __init__(self):
-    super(MoveGroupPythonInteface, self).__init__()
+    super(PathPlanningCommander, self).__init__()
     moveit_commander.roscpp_initialize(sys.argv)
     robot = moveit_commander.RobotCommander()
     scene = moveit_commander.PlanningSceneInterface()
 
     move_arm = moveit_commander.MoveGroupCommander("arm")
```

Comment:
Please make sure that this group always uses RRTstar optimized for MechanicalWork.